examples/linear_model/plot_logistic.py

```diff
@@ -57,9 +57,10 @@ def model(x):
 
 plt.ylabel('y')
 plt.xlabel('X')
-plt.xticks(())
-plt.yticks(())
+plt.xticks(range(-5, 10))
+plt.yticks([0, 0.5, 1])
 plt.ylim(-.25, 1.25)
 plt.xlim(-4, 10)
-
+plt.legend(('Logistic Curve', 'Least Squares Line'), loc="lower right",
```

Comment:
Could you label these as 
* `Logistic Regression model` and
* `Linear Regression model`